### DISCUSSION TITLE
[New Feature] Multi-Engineer immune buildings.

### DISCUSSION THREAD

**Discussion by MikillRosen** [Vinifera-Developers/Vinifera#1477] [Feature Requests]
### Description:
Now that oil derrick logic is added, it would be annoying if Multi-Engineer is on and all of a sudden you need 3 Engineers to capture an oil derrick.

### Possible Implementation:
Add `EngineerAlwaysCaptureTech` as a rules.ini INI flag and if set to yes, Engineers can capture the building no matter its health.

**Comment by MNOrac** [comment]
See also this ticket: https://github.com/Vinifera-Developers/Vinifera/issues/58
